config/selectionCfg_ttCR_Legacy2016.cfg

```diff
@@ -0,0 +1,175 @@
+[selections]
+
+# pT and eta cuts are already applied at trig match level, so they are not needed here
+baseline = ((pairType == 0) || (pairType == 1) || (pairType == 2))  && nleps == 0 && nbjetscand > 1
+
+
+# 2016 deepFlavor
+# btag requirements - DeepFlavor WPs={0.0614, 0.3093, 0.7221}; // L,M,T -- Legacy 2016 DeepFlavor
+btagMM = bjet1_bID_deepFlavor > 0.3093 && bjet2_bID_deepFlavor > 0.3093 #both jets with medium btag working point
+
+
+# updated mass cut
+invEllypsMassCut = ((tauH_SVFIT_mass-129.)*(tauH_SVFIT_mass-129.))/(53.*53.) + ((bH_mass_raw-169.)*(bH_mass_raw-169.))/(145.*145.) >  1.0
+
+
+# ABCD regions used in the analysis - DeepTau
+SR       = isOS != 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 5                           # signal region: opposite sign, isolated taus
+SStight  = isOS == 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 5                           # B region
+OSinviso = isOS != 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # C region
+SSinviso = isOS == 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 2 && dau2_deepTauVsJet < 5  # D region
```

Comment:
I think you need to take into account the pairtype here, e.g:
`SR = isOS != 0 && ((dau1_deepTauVsJet >= 5) || (dau1_iso < 0.15) || (dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 5 `
->
`SR = isOS != 0 && ((pairType == 2 && dau1_deepTauVsJet >= 5) || (pairType == 0 && dau1_iso < 0.15) || (pairType == 1 && dau1_eleMVAiso == 1)) && dau2_deepTauVsJet >= 5 `